public/javascript/canvas.js

```diff
@@ -21,7 +21,7 @@ class Canvas {
     this.ideas = [];
     this.dragging = false;
     this.selection = null;
-    this.selectionColor = "#FF5A25";
+    this.selectionColor = "transparent";
```

Comment:
Try changing it to any colour other than transparent and moving a circle - see what happens and you'll understand why I made that change! 😉